cantusdb/csv/fetch.py

```diff
@@ -0,0 +1,32 @@
+"""
+fetch the CSV exports
+"""
+
+import json
+import os
+import time
+import requests
+
+# URL of the file to download
+SOURCE_PATH = os.path.join(
+    os.path.dirname(__file__), "../data/mappings/sources_short.json"
+)
+with open(SOURCE_PATH, mode="r", encoding="utf-8") as f:
+    SOURCE_LIST = json.load(f)
```

Comment:
This isn't a constant so I think probably don't use the uppercase convention.

Reply:
Pylint tells me to use uppercase

Reply:
Pulling is wrong then. What’s the message?

Reply:
My bad, it does not return the message now, I don't know why.